Wox.Infrastructure/Hotkey/GlobalHotkey.cs

```diff
@@ -63,7 +63,7 @@ public SpecialKeyState CheckModifiers()
             }
             if ((InterceptKeys.GetKeyState(VK_WIN) & 0x8000) != 0)
             {
-                //ALT is pressed
+                //WIN is pressed
```

Comment:
hi, why you guys thinking this is helpful? are you writing code using `InterceptKeys`?